src/clue.js

```diff
@@ -117,10 +119,37 @@ var Patio = {name: 'Patio'}
 
 
 // Characters Collection
-var charactersArray = [drOrchid, drOrchid, profPlum, missScarlet, mrsPeacock, mrMustard];
+var charactersArray = [mrGreen, drOrchid, profPlum, missScarlet, mrsPeacock, mrMustard];
 
 // Rooms' Collection
 var roomsArray = [rope, knife, candlestick, dumbbell, poison, axe, bat, trophy, pistol];
 
 // Weapons Collection
 var weaponsArray = [DinningRoom, Conservatory, Kitchen, Study, Library, BilliardRoom, Lounge, Ballroom, Hall, Spa,  LivingRoom, Observatory, Theater, GuestHouse, Patio];
+
+// Iteration 2 - Creating the mystery
+
+// Random Selector
+
+function randomSelector (array) {
+  var randomCardElement = array[Math.floor(Math.random()*array.length)];
+
+  return randomCardElement; 
+} 
+
+// Create the mystery
+
+function pickMystery (array) {
+  var mysteryEnvelope = [] ;
```

Comment:
The same here and everywhere in your code you need to change all the 'mystery' for 'mystery'